src/server/components/formRunner/proxyMiddleware.ts

```diff
@@ -2,35 +2,45 @@ import { Express } from "express";
 import proxy from "express-http-proxy";
 import { FORM_RUNNER_BASE_ROUTE, FORM_RUNNER_URL } from "./constants";
 import { getFeedbackSuccessContent } from "server/components/feedback/helpers";
-import { router } from "./router";
+import { successPageContent } from "./helpers";
 
 /**
  * Proxy middleware for the form runner
  * @param app Express app
  * Important: this middleware must be added before body and cookie parsers middlewares
  */
 export function configureFormRunnerProxyMiddleware(server: Express): void {
-  server.use(router);
-
   server.use(
     `${FORM_RUNNER_BASE_ROUTE}/*`,
     proxy(FORM_RUNNER_URL, {
       proxyReqPathResolver: function (req) {
-        return req.originalUrl.replace(`${FORM_RUNNER_BASE_ROUTE}`, "");
+        return req.originalUrl.replace(FORM_RUNNER_BASE_ROUTE, "");
       },
       userResDecorator: function (_, proxyResData, userReq) {
         if (userReq.baseUrl.includes("assets/")) {
           return proxyResData;
         }
 
-        let data = proxyResData.toString("utf8");
+        let data: string = proxyResData.toString("utf8");
 
-        if (userReq.baseUrl.includes("/feedback/status")) {
-          // replace content of status page for feedback form
-          data = data.replace(
-            /(<main .*>)((.|\n)*?)(<\/main>)/im,
-            `$1${getFeedbackSuccessContent()}$4`
-          );
+        // TODO: Find a better way to do this.
+        if (userReq.baseUrl.includes("/status")) {
+          if (
+            data.search("Sorry, there is a problem with the service") === -1
+          ) {
+            if (userReq.baseUrl.startsWith("/application/feedback")) {
+              // replace content of status page for feedback form
+              data = data.replace(
+                /(<main .*>)((.|\n)*?)(<\/main>)/im,
+                `$1${getFeedbackSuccessContent()}$4`
+              );
+            } else {
+              data = data.replace(
+                /(<main .*>)((.|\n)*?)(<\/main>)/im,
+                `$1${successPageContent}$4`
```

Comment:
Does this content show up for everyone (with a successful application)? For PYE services, we do this in the dockerfile (I know you're not using docker but same idea) `COPY runner/src/server/views/ runner/dist/server/views/`, instead of having to change the response 

The PYE repo looks like this - so we're copying/replacing the forms and the views with custom forms/views. 
```
runner
└── src
    └── server
        ├── forms-v3
        │   ├── japan.json
        │   ├── morocco.json
        │   ├── portugal-import-a-vehicle.json
        │   └── thailand.json
        └── views
            ├── 500.html
            ├── application-error.html
            └── help
                └── accessibility-statement.html
```

Reply:
Good idea. I think we need to decouple the form runner from the lists application and then potentially do something like this for the overrides.